developers/compute-to-data/compute-workflow.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Compute Workflow
+section: developers
+description: Understanding the Compute-to-Data (C2D) Workflow
+---
+
+🚀  Now that we've introduced the key actors and provided an overview of the process, it's time to delve into the nitty-gritty of the compute workflow. We'll dissect each step, examining the inner workings of Compute-to-Data (C2D). From data selection to secure computations, we'll leave no stone unturned in this exploration. 
+
+For visual clarity, here's an image of the workflow in action! 🖼️✨
+
+<figure><img src="../../.gitbook/assets/c2d/c2d_detailed_flow.png" alt=""><figcaption><p>Compute detailed flow diagram</p></figcaption></figure>
+
+Below, we'll outline each step in detail 📝
+
+## Starting a C2D Job
+1. The consumer initiates a compute-to-data job by selecting the desired data asset and algorithm. 
```

Comment:
1. consumer gets the list of environments available on the provider which will be selected to start c2d job
2. consumer chooses one environment
3. The consumer initiates a compute....